From b04274406e72d8f6f7103761ef39b2655a767b43 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Wed, 4 Dec 2024 15:49:04 +0800
Subject: [PATCH 199/301] AOSCOS: platform: emc1412: mark non-prototyped
 functions as static

Fix build error with -Werror due to missing prototypes.

Fixes: "AOSCOS: MIPS: Loongson 3: Add platform device drivers"
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/platform/mips/emc1412.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/platform/mips/emc1412.c b/drivers/platform/mips/emc1412.c
index 3fb2dcd60010..bf2fa4b815c8 100644
--- a/drivers/platform/mips/emc1412.c
+++ b/drivers/platform/mips/emc1412.c
@@ -182,7 +182,7 @@ static void emc1412_shutdown(struct platform_device *dev)
  * reg1 & reg10 between (0.125, 0.875)
  * to avoid use float, temprature will mult 1000
  */
-int emc1412_internal_temp(int id)
+static int emc1412_internal_temp(int id)
 {
 	u8 reg;
 	int temp;
@@ -198,7 +198,7 @@ int emc1412_internal_temp(int id)
 	return temp;
 }
 
-int emc1412_external_temp(int id)
+static int emc1412_external_temp(int id)
 {
 	u8 reg;
 	int temp;
@@ -336,7 +336,7 @@ static void do_thermal_timer(struct work_struct *work)
 	}
 }
 
-int fixup_cpu_temp(int cpu, int cputemp)
+static int fixup_cpu_temp(int cpu, int cputemp)
 {
 	static int printed[MAX_PACKAGES] = {0, 0, 0, 0};
 	int i, value, temp_min = 50000, temp_max = -20000;
-- 
2.48.1

